content/md/first-steps/manage-your-measurements.md

```diff
@@ -0,0 +1,141 @@
+---
+id: manage-your-measurements
+themes: catalog-settings
+title: Manage your **measurements**
+popular: false
+ee-only: false
+related: what-about-measurements, manage-your-attributes
+---
+
+# See all your measurement families
+
+**Measurement families are used to store your product information with measurement i.e. weight, length or frequency inside your PIM**.    
+For more details about measurements, please read [this article](what-about-measurements.html).  
+
+To see all your measurement families, edit them or create a new measurement family, go to the `Settings/Measurements` menu:
+- *Label* is the label of a measurement family in the [UI locale](manage-your-account.html#your-interface-default-language)
+- *Code* is the identifier of a measurement family
+- *Standard unit* is the unit used to convert the other units of a measurement family
+- *Number of units* is the number of units for a measurement family (standard unit included)
+
+![List of measurement families](../img/Settings_Measurement_Families.png)
+
+You can search on the label or code of measurement families and you can sort on all the columns.
+
+# How to create a measurement family?
+
+To create a new measurement family, follow these steps:
+1.  Go to `Settings/Measurements`
+1.  Click on `Create`
+![Create a measurement family](../img/Settings_Measurement_Families_Create.png)
+1.  The pop-in to create a measurement family is displayed
+1.  Input the new measurement family code (this code is required and must be unique)
+1.  Input its label in the current [UI locale](manage-your-account.html#your-interface-default-language)
+1.  For the standard unit used to convert the other units of this measurement family, input its code (this code is required and must be unique), its label in the current [UI locale](manage-your-account.html#your-interface-default-language) and symbol
```

Comment:
```suggestion
1.  For the standard unit used to convert the other units of this measurement family, add the code (this code is required and must be unique), the label in the current [UI locale](manage-your-account.html#your-interface-default-language) and the symbol
```

Reply:
I feel like I'm missing a definition of 'symbol'

Reply:
because it's never used ;)